practices/development.md

```diff
@@ -2,105 +2,90 @@
 - **Have tasks to complete it**: Deliverables should have a set of tasks, which are actions needed to complete the deliverable.
+:::
 
-### How refinement happens
+(coordination:activity-board)=
+## The Activity Board
 
-_Deliverable Refinement_ is the process of improving the scoping, context, and structure of our Deliverables issues so that they are ready for us to work on them. When a deliverable is created, it may not have all of the information needed to take the next step. Adding that information is the goal of Deliverable Refinement.
+The [Activity Board](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) contains the collection of [Deliverables](coordination:deliverables) that the team is currently working on.
+All deliverables on the Activity Board should be accomplished by the end of the week.
+This board is filled during our [Deliverables review meeting](meetings:deliverables-review).
 
-All team members are expected to participate in deliverable refinement, though the more experienced and higher-level you are, the more you should be contributing to this process.
-The important thing is that we always have a list of high-quality deliverables ready to work towards.
+:::{admonition} The Activity Board should...
+:class: tip
+- Have enough deliverables to keep the team occupied for the week
+- Not have so many deliverables that a team member gets overwhelmed
+- Under-estimate our team's total capacity, to provide room for unexpected work (e.g., support work)
+- Have a team member assigned to each item on the board
+:::
 
-Here's a small table that explains how to decide whether deliverables issues should be put in a project backlog:
+The Activity Board is broken down into these columns:
 
-:::{list-table}
-:header-rows: 1
-- -
-  - Needs more information
-  - Ready for working
-- - Low priority
-  - Stays in issues
-  - Put in {guilabel}`Ready to work`
-- - High Priority
-  - Manually put in \
-    {guilabel}`Needs Discussion/Refinement`
-  - Put in {guilabel}`Ready to work`
-:::
+- {guilabel}`Up Next` Deliverables that are ready to be worked on.
+- {guilabel}`In progress` A deliverable that a team member is currently working towards.
+- {guilabel}`Done` Tasks that are complete! When you move a task here, make sure to update any relevant deliverables.
 
-(coordination:active-backlog)=
-## Activity Backlog
+## Requesting reviews
 
-The [Activity Backlog](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) contains the collection of **Tasks** that the team is currently working on.
-Tasks are actions that are needed to accomplish some deliverable.
-These tasks are generated from the deliverables on the Projects Backlog.
-They define a “to do” list of tasks to complete on a day-to-day basis.
+When you have an implementation that requires feedback, use GitHub's **Request Review** feature to ask other team members to take a look.
```

Comment:
In the PR description or comment, you should also mention what kind of review you want. We should also use draft PRs effectively - any team member can provide code review on PRs marked as not-a-draft, but only requested team members need to look at PRs marked as drafts. This lets us both ask for specific review early, but provide opportunity for everyone to review things before merging.

Reply:
> but provide opportunity for everyone to review things before merging.

Yep, I think it is important to give enough time (to be defined) so everyone can look at "ready" PRs before merging them (unless they are already deployed as we usually do).

Reply:
Let's tackle this one in #175 and we can update the language here as needed if the PR hasn't been merged yet?